README.md

```diff
@@ -0,0 +1,25 @@
+strong-pack
+===========
+
+Simple wrapper for `tar` for packaging a node application without ignoring
+anything except VCS files (`.git`, `.svn`, `.hg`, etc.).
```

Comment:
"without ignoring anything" -> "including everything": I had to read the double negative twice! ;-)